File: 3616-make-array-elements-equal-to-zero/make-array-elements-equal-to-zero.java
class Solution {
    public int countValidSelections(int[] nums) {
        int count = 0;

        for (int i = 0; i < nums.length; i++) {
            if (nums[i] == 0) {
                int leftSum = 0;
                for (int j = 0; j < i; j++) {
                    leftSum += nums[j];
                }

                int rightSum = 0;
                for (int j = i; j < nums.length; j++) {
                    rightSum += nums[j];
                }

                if (leftSum == rightSum) {
                    count += 2;
                } else if (Math.abs(leftSum - rightSum) == 1) {
                    count += 1;
                }
            }
        }

        return count;
    }
}

File: 3616-make-array-elements-equal-to-zero/make-array-elements-equal-to-zero.py
class Solution(object):
    def countValidSelections(self, nums):
        count=0
        for i in range (len(nums)):
            if nums[i]==0:
                if sum(nums[:i])==sum(nums[i:]):
                    count+=2
                elif sum(nums[:i])-sum(nums[i:])==1 or sum(nums[:i])-sum(nums[i:]) == -1:
                    count+=1
        return count